helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -171,3 +219,30 @@ service:
         {{- end }}
     {{- end }}
 {{- end }}
+
+{{- define "splunk-otel-collector.clusterReceiverInitContainers" -}}
```

Comment:
This looks like in a wrong place. It's being used in `helm-charts/splunk-otel-collector/templates/deployment-cluster-receiver.yaml` but defined in this config. Please move to the file where it's being used or in common. 

I'm not sure why this needs a function. May be put in the right in place? At least one condition can go away:

```
{{- if (include "splunk-otel-collector.clusterReceiverInitContainers" .) }}
```

Reply:
I had originally added a value to enable/disable like was done w/ fluentd. Inlined where used.